pom.xml

```diff
@@ -81,35 +77,37 @@
             <dependency>
                 <groupId>org.apache.httpcomponents</groupId>
                 <artifactId>httpclient</artifactId>
-                <version>${apache.http.version}</version>
+                <version>4.5.13</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.httpcomponents</groupId>
+                <artifactId>httpcore</artifactId>
```

Comment:
is this a new dependency?

Reply:
It was being pulled in implicitly, and I made it explicit because we are requesting authorization to use a specific version.